src/renderers/shared/fiber/ReactFiberBeginWork.js

```diff
@@ -266,7 +266,8 @@ module.exports = function<T, P, I, C>(config : HostConfig<T, P, I, C>) {
         // TODO: This passes the current node and reads the priority level and
         // pending props from that. We want it to read our priority level and
         // pending props from the work in progress. Needs restructuring.
-        return findNextUnitOfWorkAtPriority(workInProgress, priorityLevel);
+        var work = findNextUnitOfWorkAtPriority(workInProgress, priorityLevel);
+        return work;
```

Comment:
This change was reverted but still in your commit history due to the rebase not being able to ignore.